vault/resource_ldap_secret_backend_test.go

```diff
@@ -61,6 +61,13 @@ func TestLDAPSecretBackend(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, consts.FieldConnectionTimeout, "30"),
 				),
 			},
+			{
+				SkipFunc: func() (bool, error) {
+					return !provider.IsAPISupported(testProvider.Meta(), provider.VaultVersion116), nil
+				},
+				RefreshState: true,
+				Check:        resource.TestCheckResourceAttrSet(resourceName, consts.FieldSkipStaticRoleImportRotation),
```

Comment:
Should we set the new fields in the config and make assertions about what we expect in our state?

Reply:
oh good idea